src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs

```diff
@@ -57,6 +57,23 @@ public KustoDataProvider(OperationDataCache cache, KustoDataProviderConfiguratio
 
         public async Task<DataTable> ExecuteClusterQuery(string query, string requestId = null, string operationName = null)
         {
+            var matches = Regex.Matches(query, @"cluster\((?<cluster>([^\)]+))\).database\((?<database>([^\)]+))\)\.");
+            if (matches.Any())
+            {
+                foreach (Match element in matches)
+                {
+                    string targetCluster = element.Groups["cluster"].Value.Trim(new char[] { '\'', '\"' });
+                    string targetDatabase = element.Groups["database"].Value.Trim(new char[] { '\'', '\"' });                    
+
+                    if (!string.IsNullOrWhiteSpace(targetCluster) && !string.IsNullOrWhiteSpace(targetDatabase)
+                        && targetCluster.Equals("Aks", StringComparison.OrdinalIgnoreCase) && targetDatabase.Equals("AKSprod", StringComparison.OrdinalIgnoreCase))
```

Comment:
Will the cluster name for N-clouds?

Reply:
Not really, people write query with their public cluster name and then we make the query cloud agnostic by looking up the cluster name from Kusto mapping configuration defined by teams.

Reply:
I see. Thanks for the clarification!